sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze.py

```diff
@@ -0,0 +1,1313 @@
+                {"id": "2", "language": "es", "text": "Microsoft fue fundado por Bill Gates y Paul Allen el 4 de abril de 1975."},
+                {"id": "3", "language": "de", "text": "Microsoft wurde am 4. April 1975 von Bill Gates und Paul Allen gegründet."}]
+
+        response = client.begin_analyze(
+            docs, 
+            entities_recognition_tasks=[EntitiesRecognitionTask()], 
+            show_stats=True
+        ).result()
+
+        results_pages = list(response)
+        self.assertEqual(len(results_pages), 1)
+
+        task_results = results_pages[0].entities_recognition_results
+        self.assertEqual(len(task_results), 1)
+
+        results = task_results[0].results
+        self.assertEqual(len(results), 3)
+
+        for doc in results:
+            self.assertEqual(len(doc.entities), 4)
+            self.assertIsNotNone(doc.id)
+            #self.assertIsNotNone(doc.statistics)
+            for entity in doc.entities:
+                self.assertIsNotNone(entity.text)
+                self.assertIsNotNone(entity.category)
+                self.assertIsNotNone(entity.offset)
+                self.assertIsNotNone(entity.confidence_score)
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer(client_kwargs={
+        "api_version": TextAnalyticsApiVersion.V3_1_PREVIEW,
+        "text_analytics_account_key": os.environ.get('AZURE_TEXT_ANALYTICS_KEY'),
+        "text_analytics_account": os.environ.get('AZURE_TEXT_ANALYTICS_ENDPOINT')
+    })
+    def test_all_successful_passing_dict_pii_entities_task(self, client):
+
+        docs = [{"id": "1", "text": "My SSN is 859-98-0987."},
+                {"id": "2", "text": "Your ABA number - 111000025 - is the first 9 digits in the lower left hand corner of your personal check."},
+                {"id": "3", "text": "Is 998.214.865-68 your Brazilian CPF number?"}]
+
+        response = client.begin_analyze(
+            docs, 
+            pii_entities_recognition_tasks=[PiiEntitiesRecognitionTask()], 
+            show_stats=True
+        ).result()
+
+        results_pages = list(response)
+        self.assertEqual(len(results_pages), 1)
+
+        task_results = results_pages[0].pii_entities_recognition_results
+        self.assertEqual(len(task_results), 1)
+
+        results = task_results[0].results
+        self.assertEqual(len(results), 3)
+
+        self.assertEqual(results[0].entities[0].text, "859-98-0987")
+        self.assertEqual(results[0].entities[0].category, "U.S. Social Security Number (SSN)")
+        self.assertEqual(results[1].entities[0].text, "111000025")
+        # self.assertEqual(results[1].entities[0].category, "ABA Routing Number")  # Service is currently returning PhoneNumber here
+        self.assertEqual(results[2].entities[0].text, "998.214.865-68")
+        self.assertEqual(results[2].entities[0].category, "Brazil CPF Number")
+        for doc in results:
```

Comment:
are there also request statistics that can be verified here too?

Reply:
I think I will wait to add more validation to the statistics until the implementation is complete on the service side.  I have included tests for show_stats in the other tests above using dict as input, but I can add them here as well in the future if you think it will be more complete.